postgres/infoschema_test.go

```diff
@@ -58,6 +58,10 @@ func TestProcessInfoSchema(t *testing.T) {
 			rows: [][]driver.Value{
 				{"productid", "PRIMARY KEY"},
 				{"userid", "PRIMARY KEY"}},
+		}, {
```

Comment:
Same comment as for mysql/infoschema_test.go: Shouldn't we add a test that actually exercises the foreign key code?
This change to the test (and the following two changes) will just return empty rows for getRefTables and basically we'll never call getForeignKey. So we'll never really test the foreign key code.

Reply:
This will be done separately and I have already put it in PR note. Apologies if it's not clear.